src/libcollections/string.rs

```diff
@@ -1913,7 +1917,7 @@ impl<T: fmt::Display + ?Sized> ToString for T {
         use core::fmt::Write;
         let mut buf = String::new();
         buf.write_fmt(format_args!("{}", self))
-           .expect("a Display implementation return an error unexpectedly");
+            .expect("a Display implementation return an error unexpectedly");
```

Comment:
this change is unrelated, but it's also good; you should probably remove it since it has nothing to do with this issue.